run.ps1

```diff
@@ -168,7 +169,14 @@ function Update-Database {
     Pull-Setup
     Docker-Compose-Files
     $mssqlId = docker-compose ps -q mssql
-    docker run -it --rm --name setup --network container:$mssqlId `
+
+    # only use container network driver if using the included mssql image
+    $dockerNetworkArgs = ""
+    if (Select-String -Path ${envDir}\global.override.env -Pattern 'Data Source=tcp:mssql,1') {
```

Comment:
The searched for string in bash and powershell should match right?

```suggestion
    if (Select-String -Path ${envDir}\global.override.env -Pattern 'Data Source=tcp:mssql,1433') {
```

And also I decided to take off the `;`, technically they could have rearranged their connection string and placed it at the end without a delimiter, it would be a valid connection string but not satisfy this check, do you agree with that?

Reply:
Good points. I removed the semicolon in the shell script as well: 340e935967d03ae5410d6b28d9b979a6b93cfafb.